#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_jhead.1.dpatch by  <rousseau@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Closes: #435073: "jhead: -da example in man page reverses the dates"

@DPATCH@
--- a/jhead.1
+++ b/jhead.1
@@ -210,7 +210,7 @@
 
 Year on camera was set to 2005 instead of 2004 for pictures taken in April
 .br
-jhead -da2005:03:01-2004:03:01
+jhead -da2004:03:01-2005:03:01
 
 Default camera date is 2002:01:01, and date was reset on 2005:05:29 at 11:21 am
 .br
